sdks/csharp/graph_operations_sample.cs

```diff
@@ -1,64 +1,60 @@
 using System;
-using Azure.DigitalTwins.Core;
-using Azure.Identity;
 using System.Threading.Tasks;
 using System.IO;
 using System.Collections.Generic;
-using Azure;
-using Azure.DigitalTwins.Core.Serialization;
 using System.Text.Json;
+using Azure;
+using Azure.DigitalTwins.Core;
+using Azure.Identity;
 
 namespace minimal
 {
     class Program
     {
-
         public static async Task Main(string[] args)
         {
             Console.WriteLine("Hello World!");
 
-            //Create the Azure Digital Twins client for API calls
-            DigitalTwinsClient client = createDTClient();
+            // Create the Azure Digital Twins client for API calls
+            DigitalTwinsClient client = createDtClient();
             Console.WriteLine($"Service client created – ready to go");
             Console.WriteLine();
 
-            //Upload models
+            // Upload models
             Console.WriteLine($"Upload models");
             Console.WriteLine();
             string dtdl = File.ReadAllText("<path-to>/Room.json");
             string dtdl1 = File.ReadAllText("<path-to>/Floor.json");
-            var typeList = new List<string>();
-            typeList.Add(dtdl);
-            typeList.Add(dtdl1);
+            var models = new List<string>
+            {
+                dtdl,
+                dtdl1,
+            };
             // Upload the models to the service
-            await client.CreateModelsAsync(typeList);
+            await client.CreateModelsAsync(models);
 
-            //Create new (Floor) digital twin
-            BasicDigitalTwin floorTwin = new BasicDigitalTwin();
+            // Create new (Floor) digital twin
+            var floorTwin = new BasicDigitalTwin();
             string srcId = "myFloorID";
-            floorTwin.Metadata = new DigitalTwinMetadata();
             floorTwin.Metadata.ModelId = "dtmi:example:Floor;1";
-            //Floor twins have no properties, so nothing to initialize
-            //Create the twin
+            // Floor twins have no properties, so nothing to initialize
+            // Create the twin
             await client.CreateOrReplaceDigitalTwinAsync<BasicDigitalTwin>(srcId, floorTwin);
             Console.WriteLine("Twin created successfully");
 
-            //Create second (Room) digital twin
-            BasicDigitalTwin roomTwin = new BasicDigitalTwin();
+            // Create second (Room) digital twin
+            var roomTwin = new BasicDigitalTwin();
             string targetId = "myRoomID";
-            roomTwin.Metadata = new DigitalTwinMetadata();
```

Comment:
The type already has an object allocated, so creating a new one is not necessary